src/DynamicMvvm.Abstractions/ViewModel/IViewModel.Extensions.Commands.cs

```diff
@@ -79,6 +79,28 @@ public static partial class IViewModelExtensions
 			[CallerMemberName] string name = null
 		) => viewModel.GetOrCreateCommand(name, n => viewModel.GetDynamicCommandBuilderFactory().CreateFromTask(n, execute), configure);
 
+		/// <summary>
+		/// Gets a different Instance of a command configured for every items in a list.
+		/// </summary>
+		/// <param name="viewModel">The ViewModel.</param>
+		/// <param name="execute">The execute method.</param>
+		/// <param name="name">The command name.</param>
```

Comment:
You should specify that the name will be made unique. Trying to retrieve the command using the name specified here will not work and that must be documented.